playbooks/common/openshift-cluster/evaluate_groups.yml

```diff
@@ -37,13 +45,16 @@
       ansible_sudo: "{{ g_sudo | default(omit) }}"
     with_items: "{{ g_etcd_hosts | default([]) }}"
 
+  - set_fact:
+      g_master_hosts_to_config: "{{ g_new_master_hosts | default(g_master_hosts | default([], true), true) }}"
```

Comment:
I'd skip the set_fact here and just use `g_new_master_hosts | default(g_master_hosts | default([], true), true)` in the with_items below.